.travis.yml

```diff
@@ -4,8 +4,8 @@ env:
-  - secure: wq6H6+aa2qBhVRMZRqVtsuudVV/pZlONoVgVcJI3BRKhESw42lqHgUt/ptHxX1t26ZAjKrMmAEEGIok8qNfnTPMMurm+pa8pf/vqXiFLhb82DBvUUm38WdIvBZQ4WKrX4FeSm6aZVJJeCBsNsReQ/X4T7S2aFBgztdtkIU1JHpUt4oYhnpVIPOHsyUa8gp+wM4TqnorCdqvhlu5OdlYdAW34GnscRP89UJS9JH5M7ZyM7wV6/s+Uyav7fgV2bRjDC57tSd/OjbyMmMEHgKvvygL7qiTQm/YQFkotfVFTowxdCwzsGJwb7Qf87SShrp2IFXKsyYH3LgOUcPg3GYPBTTaVT1PCezAm63Q6WF/S2mc+VeZgDvYO6gaGNpp6v8VLGi5e36NWBX3xPAzqdZS/lxehC3BD2vE762km7arCQLm7HlFyw+QA/0m/tdOu+roarUy+2xFTy7+NH4eqP8RI7CwpknNJsapImGqAOnSh6ZWMWoIKZPIuSQdULgvNfAooSNdHUnOSx/EmCZ3e0GSLlmtVZG624k13uvvKorLBH5d0LC4wf/yeoKFdnQThgtzvvIsfs8k/TI+52+TNdUEV65nCr2834TkYUX2lG2qGU1/9IXNWT2JyZg4g+3t2nwcZbN6YYdt39PUeSnQ4kMiNHKZiRB4USyTcWxzbp6P/PB0=
-  - secure: z6WQVmKFTWf7in9J8764ic6gs5QyLctGYIDzstjSBXYrLTUfJjUwB7OAup1fAFXYUxtzkn6vjn3CIuy+5Nq7jL25Bwen4SJCPZtk9w/8acXUNDNlIFIak7T/jelOO11gHxHI6j2H+wj1BGxz1XZEkx8eeUN6389yvWgHQH01sbrQ7gXRgfmK+1pvUK9Li+4Mn9gBbOoqDxz2w8Wuazkk/FcmlYRYet4z8/y4FHwyc+Sa69utVJQAhDwJzngtb5WRdlWlrcFfyWhQ0eZ/shKxBcjekU5ZeLCh0av4ttB2IXVvLHzsr6vuCssRJqC7LHqiWEpXacWSf0ZOpMlAzwT+fPH5xXSl1kdc8vkpz8gYC5YDDMxkTrUoRv8vCYi/KWmYXNzdiAQn53aX741fcj5FV8v0ybmdPPQLfeQPDhCeHLlroWZsI5cgygxotIfeQzOY3MXSi+YqtZjmuHjAC1aINn9sLwYtCbV1ZVn3TxPaMyWjPlBuOaYoU6GkC+HxAcUy9QMVxBBzbVbOnaPri8NTXxHE58cIz+6GZVou4UDpEjjpi3Z8hieXu5cR0KKFVPofIQlgLaFZ9YgtAuXW6RP2k7ONYyCbW7bz39nnY6outFe96yR/HB4UKqBwc8dIUPfgzyoEyE1obnuNcsNf6UZf+B5ALM03ejQCx9KJuFPrZSw=
+#  - secure: wq6H6+aa2qBhVRMZRqVtsuudVV/pZlONoVgVcJI3BRKhESw42lqHgUt/ptHxX1t26ZAjKrMmAEEGIok8qNfnTPMMurm+pa8pf/vqXiFLhb82DBvUUm38WdIvBZQ4WKrX4FeSm6aZVJJeCBsNsReQ/X4T7S2aFBgztdtkIU1JHpUt4oYhnpVIPOHsyUa8gp+wM4TqnorCdqvhlu5OdlYdAW34GnscRP89UJS9JH5M7ZyM7wV6/s+Uyav7fgV2bRjDC57tSd/OjbyMmMEHgKvvygL7qiTQm/YQFkotfVFTowxdCwzsGJwb7Qf87SShrp2IFXKsyYH3LgOUcPg3GYPBTTaVT1PCezAm63Q6WF/S2mc+VeZgDvYO6gaGNpp6v8VLGi5e36NWBX3xPAzqdZS/lxehC3BD2vE762km7arCQLm7HlFyw+QA/0m/tdOu+roarUy+2xFTy7+NH4eqP8RI7CwpknNJsapImGqAOnSh6ZWMWoIKZPIuSQdULgvNfAooSNdHUnOSx/EmCZ3e0GSLlmtVZG624k13uvvKorLBH5d0LC4wf/yeoKFdnQThgtzvvIsfs8k/TI+52+TNdUEV65nCr2834TkYUX2lG2qGU1/9IXNWT2JyZg4g+3t2nwcZbN6YYdt39PUeSnQ4kMiNHKZiRB4USyTcWxzbp6P/PB0=
+#  - secure: z6WQVmKFTWf7in9J8764ic6gs5QyLctGYIDzstjSBXYrLTUfJjUwB7OAup1fAFXYUxtzkn6vjn3CIuy+5Nq7jL25Bwen4SJCPZtk9w/8acXUNDNlIFIak7T/jelOO11gHxHI6j2H+wj1BGxz1XZEkx8eeUN6389yvWgHQH01sbrQ7gXRgfmK+1pvUK9Li+4Mn9gBbOoqDxz2w8Wuazkk/FcmlYRYet4z8/y4FHwyc+Sa69utVJQAhDwJzngtb5WRdlWlrcFfyWhQ0eZ/shKxBcjekU5ZeLCh0av4ttB2IXVvLHzsr6vuCssRJqC7LHqiWEpXacWSf0ZOpMlAzwT+fPH5xXSl1kdc8vkpz8gYC5YDDMxkTrUoRv8vCYi/KWmYXNzdiAQn53aX741fcj5FV8v0ybmdPPQLfeQPDhCeHLlroWZsI5cgygxotIfeQzOY3MXSi+YqtZjmuHjAC1aINn9sLwYtCbV1ZVn3TxPaMyWjPlBuOaYoU6GkC+HxAcUy9QMVxBBzbVbOnaPri8NTXxHE58cIz+6GZVou4UDpEjjpi3Z8hieXu5cR0KKFVPofIQlgLaFZ9YgtAuXW6RP2k7ONYyCbW7bz39nnY6outFe96yR/HB4UKqBwc8dIUPfgzyoEyE1obnuNcsNf6UZf+B5ALM03ejQCx9KJuFPrZSw=
```

Comment:
Can we remove these?

Reply:
I didn't try. I was going to test that separately. This was the minimal change to unshadow the two variables we made in the travis repo settings. I don't know what the values are to put in the settings if it fails with these removed.

Reply:
OK, I did the experiment and it was fine, so I merged that change. I'll merge this.